Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-auth.tsx

```diff
@@ -198,25 +202,51 @@ const AuthProvider = ({ children }: IAuthProviderProps) => {
         setUserDetails(getUserFromResponse(authInfo));
     }, [ authInfo, getUserFromResponse, isGetAuthInfoSuccess, setUserDetails ]);
 
+    // Purpose: Added to prevent flickering Login button while loading
+    // If the user is successfully logged in > we first navigate to home
+    // then refresh the window to set internal user and header info
+    useEffect(() => {
+        if (!isLoggingIn &&
+            loginSubmitStatus === HttpStatus.Success &&
+            location.pathname === getLoginPath()) {
+            navigate('/');
```

Comment:
Clicking the login button on an empty form redirects to home

Reply:
Fixed with validations on the FE and BE